src/components/dropdown/dropdown.js

```diff
@@ -140,6 +140,7 @@ export const BDropdown = /*#__PURE__*/ Vue.extend({
       $buttonChildren = [h('span', { class: ['sr-only'] }, [this.toggleText])]
       buttonContentDomProps = {}
     }
+    const ariaHasPopupRoles = ['menu', 'listbox', 'tree', 'grid', 'dialog']
```

Comment:
That's probably not the best place to define this array but I do not know where is better.  Some constants file?